Objects/unicodeobject.c

```diff
@@ -11009,6 +11009,23 @@ _PyUnicode_Equal(PyObject *str1, PyObject *str2)
 }
 
 
+int
+PyUnicode_Equal(PyObject *str1, PyObject *str2)
+{
+    if (!PyUnicode_Check(str1)) {
+        PyErr_Format(PyExc_TypeError,
+                     "first argument must be str, not %T", str1);
```

Comment:
Maybe this should `argument %T is not str`

Reply:
The wording "X must be Y, not Z" is more common in the library. Alternatively, we have the wording "expecting X, got Y".